### PR TITLE
Bump `DESCRIPTION` version past 1.10 (`main` branch)

### PR DESCRIPTION
**Pull request by johnkerl**
Following our [established procedure](https://github.com/single-cell-data/TileDB-SOMA/wiki/Branches-and-releases)

Please see #2328 for previous

[sc-44519]

### CODE REVIEW

**Comment by johnkerl** [comment]
[sc-44519]

File: apis/r/DESCRIPTION [hunk 1]
```diff
@@ -34,7 +34,7 @@ Imports:
     Matrix,
     stats,
     bit64,
-    tiledb (>= 0.26.0),
+    tiledb (>= 0.26.0), tiledb (<= 0.26.99),
```
Comment (eddelbuettel) [inline apis/r/DESCRIPTION:37]: Yep.  If and when tiledb-r changes version and if and when that would bring in a newer TileDB Embedded we'd have to lower the '99'.   But it is a 'known unknown' now and this, even just a signal value, may help.

**Review by eddelbuettel — APPROVED** [review]
:shipit:

File: apis/r/DESCRIPTION (hunk 1 above)
Comment (johnkerl) [reply apis/r/DESCRIPTION:37]: @eddelbuettel as always a bump to core 2.23 would be accompanied by a bump of TileDB-R to 0.27 -- if a TileDB-R 0.26.anything depends someday on a core 2.23, that would be an error.

**Pull request merged**